### PR TITLE
Intrinsics

### PR DESCRIPTION
**Pull request by Lokathor**
* [x] wrap all sse (closes https://github.com/Lokathor/lokacore/issues/5)
* [x] organize the sse methods
* [x] inline(always) the sse methods
* [x] core::ops for sse
* [x] wrap all sse2 (closes https://github.com/Lokathor/lokacore/issues/6)
* [x] organize the sse2 methods
* [x] inline(always) the sse2 methods
* [x] core::ops for sse2
* [x] remove all non-temporal store/load, it's just inherently unsafe.
* [x] ~~All those From impls~~
* [x] m128
* [x] SSE top levels re-checked
* [x] SSE names re-checked
* [x] m128i
* [x] m128d
* [x] SSE2 top levels re-checked
* [x] SSE2 names re-checked
* [x] ~~sqrt_f32 and sqrt_f64 as stand alone top level functions for ease of use in no_std~~ made into an issue
* [x] ~~immediate ops?~~ issue

### CODE REVIEW

**Comment by Lokathor** [comment]
closed https://github.com/Lokathor/lokacore/issues/9

File: src/arch/sse2.rs [hunk 1]
```diff
@@ -0,0 +1,1498 @@
...
+  #[allow(clippy::trivially_copy_pass_by_ref)]
+  #[inline(always)]
+  pub fn load_high(self, new: &f64) -> m128d {
+    m128d(unsafe { _mm_loadh_pd(self.0, new) })
+  }
+
+  /// Overwrite the lower lane with the new value.
+  #[allow(clippy::trivially_copy_pass_by_ref)]
+  #[inline(always)]
+  pub fn load_load(self, new: &f64) -> m128d {
```
Comment (memoryruins) [inline src/arch/sse2.rs]:
```suggestion
  pub fn load_low(self, new: &f64) -> m128d {
```

**Comment by Lokathor** [comment]
check https://github.com/gnzlbg/typed_arch for usable content

**Comment by Lokathor** [comment]
https://github.com/gnzlbg/typed_arch may have code of value

**Pull request merged**